lib/membrane_rtc_engine/endpoints/webrtc/rtp_munger.ex

```diff
@@ -107,7 +107,7 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPMunger do
           payload_type: track.payload_type,
           marker: false,
           sequence_number: calculate_seq_num(rtp_munger.highest_incoming_seq_num + 1, rtp_munger),
-          timestamp: rtp_munger.last_timestamp
+          timestamp: rem(rtp_munger.last_timestamp + (1 <<< 32) - 100_000, 1 <<< 32)
```

Comment:
I think that the previous version was the correct one

Reply:
Actually, it was pointed out that maybe our lags are caused by the fact that we're duplicating the timestamps of the last packet, and they aren't getting pushed out of the jitter buffer in the browser. It turned out not to be the case, but I think the point of "maybe we should give them timestamps that are far in the past" isn't a bad one 🤔